source/TestFramework/CallerArgumentExpressionAttribute.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+// ReSharper disable once CheckNamespace
+namespace System.Runtime.CompilerServices
```

Comment:
Wondering if this file should belong to Test lib?

Reply:
Same as the nullable file.